src/status_im2/contexts/chat/photo_selector/style.cljs

```diff
@@ -3,28 +3,29 @@
             [react-native.platform :as platform]))
 
 (defn gradient-container
-  [safe-area]
+  [insets]
   {:width    "100%"
-   :height   (+ (:bottom safe-area) 65)
+   :height   (+ (:bottom insets) (if platform/ios? 65 85))
    :position :absolute
-   :bottom   (if platform/ios? 0 33)})
+   :bottom   0})
 
-(defn buttons-container
-  [safe-area]
-  {:flex-direction     :row
-   :justify-content    :space-between
-   :padding-horizontal 20
-   :bottom             (+ (:bottom safe-area) 33)})
+(def buttons-container
+  {:position        :absolute
+   :flex-direction  :row
+   :width           "100%"
```

Comment:
please don't specify "100%" it's redundant

Reply:
It is not redundant here.

Reply:
with absolute better to use left: 0, right: 0